industrial_ci/ros_pre-release.sh

```diff
@@ -43,17 +43,17 @@ CATKIN_DISTRO=$ROS_DISTRO
 
 case "$ROS_DISTRO" in
 "kinetic")
-    os_code_name="xenial"
+    os_code_names=(wily xenial)
     CATKIN_DISTRO="jade"
     ;;
+"indigo"|"jade")
+    os_code_names=(saucy trusty)
+    CATKIN_DISTRO="jade"
```

Comment:
CATKIN_DISTRO is just a workaround kinetic, the default ($ROS_DISTRO) is fine for jade and kinetic